<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabDashboard">
       <attribute name="title">
        <string>Dashboard</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_PTT">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Aero Matics</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(255,217,170), stop:0.5 rgb(255,187,110), stop:0.55 rgb(254,174,66), stop:1.0 rgb(254,219,116));
}

QPushButton {
        border: 1px solid rgb(5,104,204);
        border-radius: 2px;
        padding: 5px 10px 2px 5px;
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(245,249,255), stop:0.5 rgb(199,223,255), stop:0.55 rgb(175,210,255), stop:1.0 rgb(192,219,255));
        color: rgb(5,104,204);
        height: 30px;
}

QPushButton:pressed {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(192,215,255), stop:0.5 rgb(207,210,111), stop:0.55 rgb(199,223,111), stop:1.0 rgb(245,249,255));
        padding-top: 2px;
        padding-left: 3px;

}

QPushButton:on {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(90,167,45), stop:0.5 rgb(179,226,150), stop:0.55 rgb(179,226,150), stop:1.0 rgb(245,249,255));
        padding-top: 2px;
        padding-left: 3px;
}

QPushButton:disabled {
        background: transparent rgb(229,233,238);
        padding-top: 2px;        
        padding-left: 3px;
        color: red;
}</string>
          </property>
          <property name="text">
           <string>PTT</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="lineEdit_lon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
border: 1px solid;
border-radius: 5px;
border-color: rgb(5,104,204);</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_Air_Quality">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
border: 1px solid;
border-radius: 5px;
border-color: rgb(5,104,204);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
</string>
          </property>
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblNetConnectivity">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Aero Matics</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);</string>
          </property>
          <property name="text">
           <string>DASHBOARD</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lblSysDate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Aero Matics</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0,0,0);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
</string>
          </property>
          <property name="text">
           <string>Air Quality</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_Temperature">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
border: 1px solid;
border-radius: 5px;
border-color: rgb(5,104,204);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
</string>
          </property>
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
</string>
          </property>
          <property name="text">
           <string>Heart Rate (bpm)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_Heartbeat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
border: 1px solid;
border-radius: 5px;
border-color: rgb(5,104,204);</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="pushButton_Exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Aero Matics</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton:hover{
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(255,217,170), stop:0.5 rgb(255,187,110), stop:0.55 rgb(254,174,66), stop:1.0 rgb(254,219,116));
}

QPushButton {
        border: 1px solid rgb(5,104,204);
        border-radius: 2px;
        padding: 5px 10px 2px 5px;
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(245,249,255), stop:0.5 rgb(199,223,255), stop:0.55 rgb(175,210,255), stop:1.0 rgb(192,219,255));
        color: rgb(5,104,204);
        height: 30px;
}

QPushButton:pressed {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(192,215,255), stop:0.5 rgb(207,210,111), stop:0.55 rgb(199,223,111), stop:1.0 rgb(245,249,255));
        padding-top: 2px;
        padding-left: 3px;

}

QPushButton:on {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(90,167,45), stop:0.5 rgb(179,226,150), stop:0.55 rgb(179,226,150), stop:1.0 rgb(245,249,255));
        padding-top: 2px;
        padding-left: 3px;
}

QPushButton:disabled {
        background: transparent rgb(229,233,238);
        padding-top: 2px;        
        padding-left: 3px;
        color: red;
}</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButton_Shutdown">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Aero Matics</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton:hover{
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(255,217,170), stop:0.5 rgb(255,187,110), stop:0.55 rgb(254,174,66), stop:1.0 rgb(254,219,116));
}

QPushButton {
        border: 1px solid rgb(5,104,204);
        border-radius: 2px;
        padding: 5px 10px 2px 5px;
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(245,249,255), stop:0.5 rgb(199,223,255), stop:0.55 rgb(175,210,255), stop:1.0 rgb(192,219,255));
        color: rgb(5,104,204);
        height: 30px;
}

QPushButton:pressed {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(192,215,255), stop:0.5 rgb(207,210,111), stop:0.55 rgb(199,223,111), stop:1.0 rgb(245,249,255));
        padding-top: 2px;
        padding-left: 3px;

}

QPushButton:on {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.0 rgb(90,167,45), stop:0.5 rgb(179,226,150), stop:0.55 rgb(179,226,150), stop:1.0 rgb(245,249,255));
        padding-top: 2px;
        padding-left: 3px;
}

QPushButton:disabled {
        background: transparent rgb(229,233,238);
        padding-top: 2px;        
        padding-left: 3px;
        color: red;
}</string>
          </property>
          <property name="text">
           <string notr="true">Shutdown</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_lat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Swis721 Cn BT [Bits]</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(5,104,204);
border: 1px solid;
border-radius: 5px;
border-color: rgb(5,104,204);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
